packages/chatbot-server-mongodb-public/src/tracing/routesUpdateTraceHandlers.ts

```diff
@@ -126,13 +126,25 @@ function getTracingScores(
   k: number
 ) {
   return {
-    RejectedQuery: tracingData.rejectQuery === true ? 1 : null,
-    VerifiedAnswer: tracingData.isVerifiedAnswer === true ? 1 : null,
-    LlmDoesNotKnow: tracingData.llmDoesNotKnow === true ? 1 : null,
-    [`RetrievedChunksOver${k}`]:
-      tracingData.isVerifiedAnswer !== true
-        ? tracingData.numRetrievedChunks / k
-        : null,
+    // These metrics should start at 0,
```

Comment:
Not sure I follow what "start at 0" means - looks like we assign either 0 or 1 here vs assigning 0 and then later re-assigning 1.

Reply:
practically speaking, it'll always be 0 since there's never a comment/rating when the msg is first sent.